Havit.Blazor.Components.Web.Bootstrap/Grids/HxGrid.razor

```diff
@@ -259,11 +259,18 @@
 			if (totalPages > 1)
 			{
 				int currentPageIndex = CurrentUserState.PageIndex + ((CurrentUserState.LoadMoreAdditionalItemsCount + PageSizeEffective - 1) / PageSizeEffective);
-				<HxPager TotalPages="@totalPages"
-						 CurrentPageIndex="@currentPageIndex"
-						 CurrentPageIndexChanged="@HandlePagerCurrentPageIndexChanged"
-						 Settings="@PagerSettingsEffective" />
-			}
+				if (PaginationTemplate != null)
+                {
+                    @PaginationTemplate(new HxPaginationContext(currentPageIndex, totalPages, PageSizeEffective))
```

Comment:
How does the custom pagination pass the new page selected to the grid?
The `GridPaginationTemplateContext` should probably expose a `ChangeCurrentPageIndexAsync(int pageIndex)` (or similar) method (see `LoadMoreTemplate` customization below).

Also, do we want to allow changing the page size? `ChangePageSizeAsync()` method?

What it someone wants to change both in single action? Do we want to support such operation? Single `ChangePagingStateAsync(int pageIndex, int pageSize)` for both?